packages/backend/src/graphql/mutations/create-flow-transfer.ts

```diff
@@ -0,0 +1,27 @@
+import FlowTransfer from '@/models/flow-transfers'
+import Context from '@/types/express/context'
+
+type Params = {
+  input: {
+    flowId: string
+    newOwnerId: string
+  }
+}
+
+const createFlowTransfer = async (
+  _parent: unknown,
+  params: Params,
+  context: Context,
+) => {
+  const { flowId, newOwnerId } = params.input
+  const transfer: FlowTransfer = await context.currentUser
+    .$relatedQuery('sentFlowTransfers')
+    .insert({
+      flowId,
+      oldOwnerId: context.currentUser.id,
+      newOwnerId,
+    })
+  return transfer.id // TODO (mal): change to boolean if not needed
```

Comment:
nothing wrong with returning the entire `transfer` object, we can just make our front end query only the `id` field.

The network usage is not increased because Apollo server will only return the queried fields.